### PR TITLE
Add a metric file describing the service checks state

### PR DESCRIPTION
**Pull request by Aestek**

### CODE REVIEW

File: samples/service_checks_metrics.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# A template for exporting metrics for prometheus using consul-templaterb
+# This template can be configure the following way with environment variables
+# Environment variables to filter services/instances
+#   SERVICES_TAG_FILTER: basic tag filter for service (default HTTP)
+#   INSTANCE_MUST_TAG: Second level of filtering (optional, default to SERVICES_TAG_FILTER)
+#   INSTANCE_EXCLUDE_TAG: Exclude instances having the given tag
+#   EXCLUDE_SERVICES: comma-separated services regexps to exclude (example: lbl7.*,netsvc-probe.*,consul-probed.*)
```
Comment (pierresouchay) [inline samples/service_checks_metrics.erb]: Those variables do not work

Comment (pierresouchay) [reply samples/service_checks_metrics.erb]: (And those variables should work)

File: samples/service_checks_metrics.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+
+# HELP consul_service_check_count The number of check for each service
+# TYPE consul_service_check_count gauge
+
+<%
+  services().each do |service_name, tags|
+    service(service_name).each do |snode|
+%>consul_service_check_count{service="<%= service_name %>", node="<%= snode['Node']['Node'] %>"} <%= snode['Checks'].select{|chk| chk['ServiceID'] != ""}.length %>
```
Comment (pierresouchay) [inline samples/service_checks_metrics.erb]: It might be interesting to check services not having serfHealth as well

Comment (Aestek) [reply samples/service_checks_metrics.erb]: Done, I've added consul_node_check_count

**Pull request merged**